### ISSUE TITLE
Support setting initial_cluster_state for etcd

### ISSUE THREAD

**Issue by clly** [puppetlabs/puppetlabs-kubernetes#245]
Support configuring the intial_cluster_state for etcd. 

Why: etcd does not come up gracefully if it is started on a server that was not originally part of the cluster. In order to add the node the initial_cluster_state flag needs to be set to "existing".
What: Add a flag to the kubernetes module to support setting this etcd flag.

**Comment by temujin9** [comment]
@davejrt Can you tag this? 3.2.1 seems appropriate.

**Comment by davejrt** [comment]
Yep that's the plan, just need to get someone from my team to review
